docs/src/components/IconsList.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import {
+  Button,
+  Card,
+  CardSection,
+  Coupon,
+  Heading,
+  InputField,
+  Stack,
+  Text,
+  Tile,
+} from "@kiwicom/orbit-components";
+import { Link as LinkIcon, Search } from "@kiwicom/orbit-components/icons";
+import iconsList from "@kiwicom/orbit-components/lib/data/icons.json";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+import { CodeBlock } from "./Code";
+import { copyTimeout, pascalize } from "../utils/common";
+
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
+  grid-gap: ${({ theme }) => theme.orbit.spaceXSmall};
+`;
+
+const StyledAnchorWrapper = styled.a`
+  a {
+    float: left;
+    padding-top: 4px;
+    padding-right: 2px;
+    margin-left: -22px;
+  }
+
+  svg {
+    height: 20px;
+    width: 20px;
+    visibility: hidden;
+  }
+
+  &:hover svg {
+    visibility: visible;
+  }
+`;
+
+interface IIconObject {
+  character: string;
+  description: string;
+  svg: string;
+  url: string;
+}
+
+interface IconProps {
+  iconName: string;
+  opened: boolean;
+  setOpenedIcon: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const Icon = ({ iconName, opened, setOpenedIcon }: IconProps) => {
+  const [copied, setCopied] = useState(false);
+  useEffect(() => {
+    copyTimeout(copied, setCopied);
+  }, [copied, setCopied]);
+
+  const iconObject: IIconObject = iconsList[iconName];
+  const pascalName = pascalize(iconName);
+
+  return (
+    <div id={iconName}>
+      <Tile onClick={() => setOpenedIcon(opened ? "" : iconName)}>
+        <Stack justify="between">
+          <img src={`data:image/svg+xml;utf8,${iconObject.svg}`} alt="" />
+          <Coupon>{iconObject.character}</Coupon>
+        </Stack>
+        <Text type="secondary">{iconName}</Text>
+      </Tile>
+      {opened && (
+        <Card>
+          <CardSection>
+            <Stack>
+              <StyledAnchorWrapper>
+                <Heading as="h3" type="title2" spaceAfter="medium">
+                  <a href={`#${iconName}`}>
+                    <LinkIcon />
+                  </a>
+                  {iconName}
+                </Heading>
+              </StyledAnchorWrapper>
+              {iconObject.description && <Text>{iconObject.description}</Text>}
+              <CodeBlock className="language-jsx">{`import { ${pascalName} } from "@kiwicom/orbit-components/icons"\n\n<${pascalName} />`}</CodeBlock>
```

Comment:
when i opened it locally, code aligned on the right side, should'n it be aligned on the left? 🤔

Reply:
I see it left aligned, both locally and at https://orbit-docs-docs-iconslist.surge.sh/foundation/icons/#clock. What browser are you using?

Reply:
Ofc Chrome. I think it's because i used it on the wrong place on the main page to look on it (it could be affected by Stack or other wrapper), will check it again